demos/common/cpp/models/include/models/detection_model_yolo.h

```diff
@@ -71,5 +82,14 @@ class ModelYolo : public DetectionModel {
     std::map<std::string, Region> regions;
     double boxIOUThreshold;
     bool useAdvancedPostprocessing;
-    bool isYoloV3;
+    YoloVersion yoloVersion;
+    const std::vector<float> presetAnchors;
+    const std::vector<int64_t> presetMasks;
+
+    static float sigmoid(float x) {
+        return 1.f / (1.f + exp(-x));
+    }
+    static float linear (float x) {
+        return x;
```

Comment:
Should not be a class member, should be only in `*.cpp` file. Or even replaced by lambdas

Reply:
why ? it is protected and not visible outside. We should minimize number of functions in global scope

Reply:
If you move them to *cpp they won't be visible outside either.

Reply:
done